tests/backends/test_ddl_references.py

```diff
@@ -5,12 +5,21 @@
 
 
 class TableTests(SimpleTestCase):
-    reference = Table('table', lambda table: table.upper())
+    def setUp(self):
```

Comment:
I guess you might as well use setUp from the beginning to avoid extra code changes.